lib/Pod/To/HTML.pm

```diff
@@ -157,17 +158,48 @@ sub assemble-list-items(:@content, :$node, *% ) {
 
 
 #| Converts a Pod tree to a HTML document.
-sub pod2html($pod, :&url = -> $url { $url }, :$head = '', :$header = '', :$footer = '', :$default-title,
-  :$css-url = '//design.perl6.org/perl.css', :$lang = 'en',
+sub pod2html(
+    $pod,
+    :&url = -> $url { $url },
+    :$head = '',
+    :$header = '',
+    :$footer = '',
+    :$default-title,
+    :$css-url = '//design.perl6.org/perl.css',
+    :$templates,
+    :$lang = 'en'
     --> Str ) is export {
+
+    my $template-dir = 'resources/templates';
```

Comment:
And this should probably be an argument, with this value as a default. Don't like too much hardcoded paths (or mostly anything).

Reply:
This is actually wrong. It works in a test environment in a module, but it does not work when 'wild'. The resource needs to be in referenced in META6.json and then in %?RESOURCES. I will change.